File: C/Lista_02(Estrutura_Condicional)/Tempo_de_Jogo.c
#include <stdio.h>

int main () {

    int horaInicial, horaFinal;

    printf("Hora inicial: ");
    scanf("%d", &horaInicial);

    printf("Hora final: ");
    scanf("%d", &horaFinal);

    if (horaFinal<=horaInicial){

        horaFinal = horaFinal + 24;

    }

    int duracao = horaFinal - horaInicial;

    printf("O JOGO DUROU %d HORA(S)", duracao);

    return 0;

}

File: Python/Lista_02(Estrutura_Condicional)/Tempo_de_Jogo.py
horaInicial = int(input("Hora inicial: "))

horaFinal = int(input("Hora final: "))

if horaInicial>=horaFinal:
    horaFinal = horaFinal + 24

duracao = horaFinal-horaInicial

print(f"O JOGO DUROU {duracao} HORA(S)")